packages/TelegramClient-Core.package/TCCCore.class/instance/imageStore..st

```diff
@@ -0,0 +1,5 @@
+accessing
+imageStore: anImageStore
+
+	imageStore := anImageStore.
+	anImageStore core: self.
```

Comment:
I think we copied the approach from the setter of the userStore (users:). In that setter the core is set, because some test uses it. Do you think we should remove the line in both setters because it is too defensive? Or is it worth keeping to avoid nasty bugs where a store doesn't have the correct core?

Reply:
We can keep it but should add a comment why

Reply:
We should also clearly define when to use this pattern and who sets themselves on whom. Why does the core set itself on the store, and not the other way around?

Reply:
All our tests are running without problems so we deleted the duplicate setter